backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/TableResource.kt

```diff
@@ -0,0 +1,18 @@
+package pl.poznan.put.tsd.planningpoker.backend.resources
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import pl.poznan.put.tsd.planningpoker.backend.resources.responses.TableCreatedResponse
+import pl.poznan.put.tsd.planningpoker.backend.services.GamesService
+
+@RestController
```

Comment:
please add `/poker-api/v1` prefix to the Path (or maybe `/poker-api/v1/table-api`)